src/System.Diagnostics.PerformanceCounter/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
```

Comment:
@danmosemsft we are still going to need configurations for netcoreapp, see https://github.com/dotnet/corefx/issues/24776, do you want to do those all in one pass or as folks are working on the library?

Reply:
I guess we should try to catch this in PR's then do a pass for all the rest.

Reply:
I think it's fine for @adiaaida to submit this without and @safern will fix it in his pass.

Reply:
We will still need to do a pass to make sure everything is marked as inbox correctly for Xamarin frameworks once we talk to @marek-safar next week that he'll be around. So it's ok if I take that as part of that pass :)

Reply:
OK I'm fine with doing it in one pass.